### PR TITLE
Add Test Connection button to JDBC user stores

### LINKED ISSUE #649: JDBC User Stores should have a Test Connection button
**Describe the Issue:**
JDBC user stores should have a button to test the connection in both the add wizard and the edit view. The user should also be able to add a user store even if the connection fails.

### PR DESCRIPTION
**Pull request by thivi**
image
image

### CODE REVIEW

File: apps/admin-portal/src/components/user-stores/edit/edit-connection-details-user-store.tsx [hunk 1]
```diff
@@ -203,9 +193,39 @@ const EditConnectionDetails = (
...
             <Grid.Row columns={ 1 }>
                 <Grid.Column width={ 8 }>
-                    <PrimaryButton onClick={ ()=>{setSubmit()} }>
+                    <PrimaryButton onClick={ () => { setSubmit() } }>
                         Update
                     </PrimaryButton>
+                    <LinkButton
+                        onClick={
+                            () => {
+                                if (type.typeId === "SkRCQ1VzZXJTdG9yZU1hbmFnZXI") {
```
Comment (NipuniBhagya) [inline apps/admin-portal/src/components/user-stores/edit/edit-connection-details-user-store.tsx:202]: Shall we assign this to constant and use it here?

File: apps/admin-portal/src/components/user-stores/wizards/basic-details-user-store.tsx [hunk 2]
```diff
@@ -16,11 +16,12 @@
 * under the License.
 */
 
-import React, { useState, useEffect } from "react";
-import { Forms, FormValue, Field } from "@wso2is/forms";
-import { TypeResponse, AlertLevels } from "../../../models";
-import { getTypes } from "../../../api";
+import { AlertLevels, TypeResponse } from "../../../models";
+import { Field, Forms, FormValue } from "@wso2is/forms";
+import React, { useEffect, useState } from "react";
```
Comment (NipuniBhagya) [inline apps/admin-portal/src/components/user-stores/wizards/basic-details-user-store.tsx:21]: Let's get this import to the top

File: apps/admin-portal/src/components/user-stores/wizards/connection-details.tsx [hunk 3]
```diff
@@ -16,12 +16,13 @@
...
 
-import React, { useEffect, useState } from "react";
-import { Field, FormValue, Forms } from "@wso2is/forms";
-import { TypeProperty, Type, AlertLevels, TestConnection } from "../../../models";
-import { Grid, Message } from "semantic-ui-react";
+import { AlertLevels, TestConnection, Type, TypeProperty } from "../../../models";
+import { Divider, Grid, Message } from "semantic-ui-react";
+import { Field, Forms, FormValue } from "@wso2is/forms";
 import { getAType, testConnection } from "../../../api";
+import React, { useEffect, useState } from "react";
```
Comment (NipuniBhagya) [inline apps/admin-portal/src/components/user-stores/wizards/connection-details.tsx:23]: Let's get this to the top

File: apps/admin-portal/src/pages/claim-dialects.tsx [hunk 4]
```diff
@@ -187,16 +195,16 @@ export const ClaimDialectsPage = (): React.ReactElement => {
                                                     centered
                                                     size="mini"
                                                 >
-                                                    <ClaimsAvatarBackground primary/>
+                                                    <ClaimsAvatarBackground primary />
                                                     <span className="claims-letter">
-                                                L
+                                                        L
```
Comment (NipuniBhagya) [inline apps/admin-portal/src/pages/claim-dialects.tsx:200]: Check if there's an alignment issue here

File: apps/admin-portal/src/components/user-stores/wizards/basic-details-user-store.tsx (hunk 2 above)
Comment (thivi) [reply apps/admin-portal/src/components/user-stores/wizards/basic-details-user-store.tsx:21]: Eslint doesn't allow it. Imports have to be alphabetically sorted.

**Pull request merged**